plugins/woocommerce/tests/e2e-pw/README.md

```diff
@@ -93,50 +94,44 @@ Edit [.wp-env.json](https://github.com/woocommerce/woocommerce/blob/trunk/plugin
 
 **Modiify port for e2e-environment**
 
-Edit [tests/e2e/config/default.json](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/tests/e2e/config/default.json).****
+Edit [tests/e2e/config/default.json](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/tests/e2e/config/default.json).\*\*\*\*
 
 ### Starting/stopping the environment
 
 After you run a test, it's best to restart the environment to start from a fresh state. We are currently working to reset the state more efficiently to avoid the restart being needed, but this is a work-in-progress.
 
-- `pnpm env:down --filter=woocommerce` to stop the environment
-- `pnpm env:destroy --filter=woocommerce` when you make changes to `.wp-env.json`
+-   `pnpm env:down --filter=woocommerce` to stop the environment
+-   `pnpm env:destroy --filter=woocommerce` when you make changes to `.wp-env.json`
 
```

Comment:
Looping in @nigeljamesstevenson on this discussion as well, since we may want to consider updating the API tests documentation to align with these changes too.

Reply:
Thanks for looping me on on this @zhongruige - i'll update the API tests documentation to align with these changes!

Reply:
> Also, would it make sense to rename `env:test` to `env:start`? This would be aligned with the `wp-env` vocabulary, just as down and destroy do.

Cross referencing with the [package.json](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/package.json), if we rename `env:test` to `env:start` then it could be confused with `env:dev` which in turn calls `pnpm wp-env start`

we can also see that `pnpm env:down` in turn calls `pnpm wp-env stop` so it would be better to keep these consistent

Currently, the following could be updated:
`"env:dev": "pnpm wp-env start",` - could be changed to `"env:start": "pnpm wp-env start",`
`"env:test": "pnpm run env:dev && ./tests/e2e-pw/bin/test-env-setup.sh",` - could then be changed to `"env:test": "pnpm run env:start && ./tests/e2e-pw/bin/test-env-setup.sh",` (this would differentiate between the dev and test)
`"env:down": "pnpm wp-env stop",` - could be changed to `"env:stop": "pnpm wp-env stop", `

Whatever updates are made, I'll make sure the equivalent is updated for the `api-core-tests`
		
Thanks @tammullen @alopezari @zhongruige @lanej0 !

Reply:
You're right @nigeljamesstevenson! I forgot about `pnpm env:dev`.

Maybe we can just do `pnpm env:dev:start` to start the dev environment and `pnpm env:test:start` to start the test one. And also apply the related changes to stop and destroy.

I just suggested this because if you read `pnpm env:test` out of context, you might think that this command "tests" something, but it just starts the test environment.

This might make the structure of the commands a bit more complex, so I'd like to know your thoughts here too @nigeljamesstevenson @tammullen @zhongruige @lanej0, in case I'm missing something and complicating things 😄

Reply:
Perhaps we could go with something simple like this?

`pnpm env:up`

Reply:
We would be in a similar situation, isn't it? We should differentiate between dev and test, so it would look like
`pnpm env:dev:up` and `pnpm env:test:up`

Reply:
Hmm I feel like chaining that way makes it a bit more complex, I am not sure if we need the `dev` in there. So if we just had the following:

* `pnpm env:start`
* `pnpm env:stop`
* `pnpm env:destroy`

Perhaps to avoid the `env:test` issue, maybe we just rename this to something like `pnpm env:setup` (or configure or something like that) since it just [sets up the environment](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/tests/e2e-pw/bin/test-env-setup.sh) required for the tests. This way everything environment related is under `env`.

With the above, then we have our commands for the tests ran against the env and for any new tests, since it's a bit more semantically clear that these run tests and nothing specifically with the env:

* `pnpm test:e2e-pw`
* `pnpm test:api-pw`
* `pnpm test:k6`

Reply:
I like this!

If we don't need to keep the dev environment in mind for this naming convention, then we can avoid that extra layer of complexity in the command, so as you said, this would look like:
```
pnpm env:start
pnpm env:stop
pnpm env:destroy
```
Or even replace `pnpm env:start` with `pnpm env:setup`, as you suggested, both look fine to me 👍 

Thanks @zhongruige!

Reply:
I agree - the less complexity the better!

```
pnpm env:start
pnpm env:stop
pnpm env:destroy
```
which all tie in nicely with the `wp-env` commands that they are calling in the package.json and then also have
`pnpm env:setup`
for the setting up of the test environment i.e. this starts the environment with `pnpm env:start` and then calls the `test-env-setup.sh` for the additional test env setup

i.e. I would keep `pnpm env:start` and not replace it 😊

Thanks @alopezari  and @zhongruige 👏🏻